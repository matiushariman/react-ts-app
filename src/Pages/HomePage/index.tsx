import * as React from 'react';

import Heading from 'components/Heading';

export function HomePage(): React.ReactElement {
  return <Heading>HomePage</Heading>;
}

export default HomePage;
