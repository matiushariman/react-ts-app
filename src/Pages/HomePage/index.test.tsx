import React from 'react';
import { cleanup, render } from '@testing-library/react';

import HomePage from '.';

afterEach(cleanup);

test('renders the component', () => {
  const { container } = render(<HomePage />);

  expect(container).toMatchSnapshot('SecondaryPage');
});
