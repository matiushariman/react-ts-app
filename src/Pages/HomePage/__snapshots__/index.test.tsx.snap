// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the component: SecondaryPage 1`] = `
<div>
  <h1
    class="sc-AxjAm hgJWxN"
  >
    HomePage
  </h1>
</div>
`;
